lib/ssl/src/ssl_gen_statem.erl

```diff
@@ -1161,6 +1143,16 @@ handle_alert(#alert{level = ?WARNING, description = ?NO_RENEGOTIATION} = Alert,
     %% Go back to connection!
     State = Connection:reinit(State0#state{handshake_env = HsEnv#handshake_env{renegotiation = undefined}}),
     Connection:next_event(connection, no_record, State);
+handle_alert(#alert{level = ?WARNING, description = ?USER_CANCELED} = Alert, StateName,
+	     #state{static_env = #static_env{role = Role,
+                                             protocol_cb = Connection},
+                    connection_env = #connection_env{negotiated_version = Version},
```

Comment:
`Version -> _Version` ?

Reply:
Version is not needed, will remove it. Needs merge of other branches before testing properly.